.github/workflows/ci.yml

```
@@ -0,0 +1,66 @@
+name: CI
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-latest-16-cores
```

```suggestion
    runs-on: ubuntu-latest
```

We don't have large runners yet.